### PR TITLE
Try to create better card view

### PR DESCRIPTION
**Pull request by sviatoslav-krupa**
**CHECK LIST**
- [ ] specs added
- [x] specs passed
- [x] code coverage >90%
- [ ] DB indexes added if needed
- [ ] seed.rb file is updated if needed
- [x] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [ ] I've checked new feature as logged in and logged out user if needed
- [x] "let!" is used in specs
- [ ] screenshot of rails_best_practices and other tools added
- [x] screenshot of feature is added if task has UI

screen

### CODE REVIEW

File: app/controllers/account/pages_controller.rb [hunk 1]
```diff
@@ -35,12 +36,15 @@ def collection
     parent.pages
   end
 
-  def list_without_params
-    collection.by_rating.page(params[:page]).per(9)
+  def list_without_params(per_page)
+    collection.by_rating.page(params[:page]).per(per_page)
```
Comment (igorkasyanchuk) [inline app/controllers/account/pages_controller.rb]: just put `.per(12)`

and remove this param

File: app/controllers/account/pages_controller.rb [hunk 2]
```diff
@@ -35,12 +36,15 @@ def collection
...
   end
 
-  def list_with_params
-    Page.search(params[:q], where: { company_id: parent.id }, page: params[:page], per_page: 9)
+  def list_with_params(per_page)
+    Page.search(params[:q],
+                where: { company_id: parent.id },
+                page: params[:page],
+                per_page: per_page)
```
Comment (igorkasyanchuk) [inline app/controllers/account/pages_controller.rb]: per_page: 9

Comment (igorkasyanchuk) [reply app/controllers/account/pages_controller.rb]: or even LIST_VIEW_PER_PAGE = 9 as constant

File: app/controllers/account/pages_controller.rb [hunk 3]
```diff
@@ -1,7 +1,8 @@
 class Account::PagesController < ApplicationController
   def index
     @partial = whitelisted_partial || 'pages_list'
-    @pages = params[:q].blank? ? list_without_params : list_with_params
+    per_page = @partial == 'pages_list' ? 10 : 12
```
Comment (igorkasyanchuk) [inline app/controllers/account/pages_controller.rb]: remove this line (see below why)

File: app/views/account/pages/_pages_card.html.slim [hunk 4]
```diff
@@ -1,15 +1,20 @@
 .row
   - pages.each do |page|
-    .col-4
+    .col-3
       .card.card-view
-        - if page.screenshot?
+        .card-content
+          .card-body
+            .header
+              h4.card-title = page.title
```
Comment (igorkasyanchuk) [inline app/views/account/pages/_pages_card.html.slim]: what if title is very long?

**Comment by sviatoslav-krupa** [comment]
new_text

**Pull request merged**